src/test/osd/osd-scrub-repair.sh

```diff
@@ -925,7 +927,7 @@ function TEST_corrupt_scrub_replicated() {
 EOF
 
     jq "$jqfilter" $dir/json | python -c "$sortkeys" | sed -e "$sedfilter" > $dir/csjson
-    diff -y $termwidth $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
+    diff ${DIFFCOLOPTS} $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
```

Comment:
@tchaikov 
Ah, oke, we are on the smae page then.

Found another usage of the same type of problem.
If we do the lifting, I suggest putting it in ceph-helper.sh, but that will require a full QA run and sorts. Not sure if it is worth the effort.
I've now repaired it like I did before.